k3k-kubelet/provider/provider.go

```diff
@@ -42,7 +48,7 @@ type Provider struct {
 	VirtualClient    client.Client
 	ClientConfig     rest.Config
 	CoreClient       cv1.CoreV1Interface
-	MetricsClient    metricset.Interface
+	MetricsClient    metricset.Interface // TODO: do we need this?
```

Comment:
I think it was just copied from the virtual-kubelet definition, I think we should remove it until we use it

Reply:
Yes, it's just an interface that can be implemented if needed. Probably useful for testing, but not necessary at the moment.